05-tips.Rmd

```diff
@@ -45,7 +45,7 @@ Use
 
     git reset --hard <commit-hash>
 
-where <commit-hash> is the unique hash of the commit you want to go back to (you can find this via `git log`).
+where `<commit-hash>` is the unique hash of the commit you want to go back to (you can find this via `git log`).
```

Comment:
I think same here as previous comment, this should probably be 'commit-hash' since the <> aren't needed